lib/resources/file.rb

```diff
@@ -35,10 +35,11 @@ class FileResource < Inspec.resource(1)
     "
 
     attr_reader :file, :mount_options
-    def initialize(path)
+    def initialize(path, sensitive = false)
```

Comment:
I feel that we may do not want to add an additional option to each resource. This adds additional burden and makes resource development more complex.